### PR TITLE
Build a Bank App - Added  Node.js Course By Ugwu Somto

### PR DESCRIPTION
**Pull request by ugwusomto**
## What does this PR do?
Add resource(s)

## For resources
It is a Course

### Description
It is a Node.js Project-based Course

### Why is this valuable (or not)?
Yes it is valuable

### How do we know it's really free?
It's on Youtube

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by ugwusomto** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

I have updated the URL to match the playlist url

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1082,6 +1082,7 @@
 
 * [A Beginner's Guide to Node.js](https://www.udemy.com/course/a-beginners-guide-to-nodejs) - DSC VIT Powered by Google Developers, Md Hishaam Akhtar (Udemy)
 * [Beginner's Series to: Node.js](https://www.youtube.com/playlist?list=PLlrxD0HtieHje-_287YJKhY8tDeSItwtg) - Microsoft Developer
+* [Build a Full Bank Application with Node.js](https://www.youtube.com/playlist?list=PL8YhozOQUteYwa13S-PGUNMM37gv7gEVq) - Ugwu Somto
```
[the remark this answers is not in the document]
Comment (ugwusomto) [reply courses/free-courses-en.md]: I have updated the course name to match what exist in the course on youtube

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**